apps/analysis/models.py

```diff
@@ -0,0 +1,73 @@
+from django.db import models
+from django.contrib.postgres.fields import JSONField
+
+from user.models import User
+from project.models import Project
+from entry.models import Entry
+
+
+class Analysis(models.Model):
+    title = models.CharField(max_length=255)
+    team_lead = models.ForeignKey(
```

Comment:
Include project id as well
@thenav56 this is where we include it right?

Reply:
Yes.